File: arrays/csharp/find_even_number_digits.cs
using System;

public class find_even_number_digits {

    static int FindNumbers(int[] nums) {
        int countEvenNumber = 0;
        foreach (int value in nums)
        {
            int numDigits = value.ToString().Length;
            if (numDigits % 2 == 0)
            {
                countEvenNumber++;
            }
        }
        return countEvenNumber;
    }

    static void Main() {
        Console.WriteLine(FindNumbers(new int[] { 12, 345, 2, 6, 7896 }));  // 2
        Console.WriteLine(FindNumbers(new int[] { 555, 901, 482, 1771 }));  // 1
    }
}

File: arrays/java/find_even_number_digits.java
public class find_even_number_digits {

    public static int findNumbers(int[] nums) {
        int countEvenNumber = 0;
        for (int value : nums) {
            int numDigits = String.valueOf(value).length();
            if (numDigits % 2 == 0) {
                countEvenNumber++;
            }
        }
        return countEvenNumber;
    }

    public static void main(String[] args) {
        System.out.println(findNumbers(new int[]{12, 345, 2, 6, 7896}));  // 2
        System.out.println(findNumbers(new int[]{555, 901, 482, 1771}));  // 1
    }
}
